src/graphql/helpers.ts

```diff
@@ -53,6 +54,13 @@ export function checkLimits(args: any = {}, type) {
   const { where = {} } = args;
   const typeLimits = { ...ARG_LIMITS.default, ...(ARG_LIMITS[type] || {}) };
 
+  // overwrite default limit for all *_in fields
```

Comment:
We should also limit the `_not_in` fields

Reply:
Looks like it already limits `_not_in` 😅 

```suggestion
  // overwrite default limit for all *_in or *_not_in fields
```